spec/features/snap_application_with_maximum_info_spec.rb

```diff
@@ -0,0 +1,250 @@
+      within(find(:fieldset, text: "Who is enrolled in college?")) do
+        check "Joey"
+      end
+
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      choose "Yes"
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      fill_in "step[income_change_explanation]", with: "EXPLAINING!!!"
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      select_employment(
+        full_name: "Jessie Tester",
+        employment_status: "Self Employed",
+      )
+      select_employment(
+        full_name: "Joey Tester",
+        employment_status: "Employed",
+      )
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      fill_in "Employer name", with: "Company ABC Inc."
+      fill_in "Usual hours per week", with: 25
+      fill_in "Pay (before tax)", with: 100
+      select "Daily", from: "How often are you paid that amount?"
+
+      fill_in "Type of work", with: "Chef"
+      fill_in "Monthly pay (before tax)", with: 300
+      fill_in "Monthly expenses", with: 100
+
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      check "Pension"
+      check "Social Security"
+
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      fill_in "Pension", with: "100"
+      fill_in "Social Security", with: "5"
+
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      choose_yes("Does your household have any money or accounts?")
+      choose_yes("Does your household own any property or real estate?")
+      choose_yes("Does your household own any vehicles?")
+
+      click_on "Continue"
+    end
+
+    on_page "Money & Income" do
+      fill_in(
+        "In total, how much money does your household have in cash and \
+accounts?",
+        with: 100,
+      )
+      check "Other"
+
+      click_on "Continue"
+    end
+
+    on_page "Expenses" do
+      click_on "Continue"
+    end
+
+    submit_housing_expenses
+    submit_expense_sources(answer: "yes")
+    submit_expenses
+
+    on_page "Contact Preferences" do
+      choose_yes(
+        "Would you like text message reminders about key steps and documents" +
+        " required to help you through the enrollment process?",
+      )
+      click_on "Continue"
+    end
+
+    on_page "Contact Preferences" do
+      click_on "Continue"
+    end
+
+    on_page "Other Details" do
+      fill_in "Anything else?", with: "This is helpful, thank you!"
+      click_on "Continue"
+    end
+
+    consent_to_terms
+
+    on_page "Sign and Submit" do
+      fill_in "Your signature", with: "Jessie Tester"
+      click_on "Sign and submit"
+    end
+
+    on_page "Documents" do
+      upload_documents
+      click_on "Done uploading documents"
+    end
+
+    on_page "Application Submitted" do
+      fill_in "Your email address", with: "test@example.com"
+      click_on "Submit"
+    end
+
```

Comment:
__not blocking__ - Thoughts on checking that the snap_application.pdf_attributes has all the data you expect here?

Reply:
Part of me would like to, but I am hesitant to check for data in feature specs (prefer to keep them at the level of what a person would see). We do have pretty decent controller specs coverage for things like data. What do you think, how do you usually handle things like this?

Reply:
Yeah, I think that's valid. I think it's gonna be really hard to test what someone sees when it comes to the PDF; which is probably OK.